Resources/Prototypes/SS220/Entities/Markers/Spawners/jobs.yml

```
@@ -1,15 +1,20 @@
+# Follow the alphabetical order of.
+# Следуйте алфавитному порядку.
+
+# Адвокат
 - type: entity
-  id: SpawnPointSeniorEngineer
+  id: SpawnPointActualFuckingLawyer
```

```suggestion
  id: SpawnPointActualLawyer
```
Это разве нельзя так сделать?

---

Придётся прототипы карт менять, оно нам надо? 0.0